src/apps/api/tests/test_queue_api_permissions.py

```diff
@@ -4,15 +4,15 @@
 from django.test import TestCase
 from django.urls import reverse
 
-from factories import UserFactory
+from factories import UserFactory, QueueFactory
 from queues.models import Queue
 
 
 class QueuesAPITestCase(TestCase):
```

Comment:
overall, in all these tests, we should probably assert what status codes _are_ rather than what they _are not_. Maybe something changes and something that previously returned a 201 now just returns a 200, and we want it it return a 403. Asserting `response.status_code != 201` would still pass with a `200` response when it should fail.